src/Components/ActivityCards/ActivityCards.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { RiMovie2Line , RiMusic2Line} from "react-icons/ri";
+import { FaPodcast, FaChessKnight} from "react-icons/fa";
+import { CgCardHearts } from "react-icons/cg";
+import './ActivityCards.scss'
+export const ActivityCards = () => {
```

Comment:
I really like this strategy of having this `ActivityCards` component for more modular organization.  Maybe, though, we should change the name of the component to something that is more indicative of what is being rendered.  I can't think of an example now, but maybe we can brainstorm together if the team agrees!

Reply:
Yeah we can brainstorm some conventions as I really don't whats best practice for naming components, like `MovieView` or `MovieResult` no idea!

Reply:
For sure, I am open to any suggestions.